images/x86_64/Ubuntu_14.04/shippable.yml

```diff
@@ -2164,6 +2164,56 @@ jobs:
       script:
         - cat output.txt
 
+  # NOTE: This is a temporary job and is not meant to be complete to be used
+  # as an IN for other jobs.
+  - name: v582_u14_x8664_k313_gce_img
+    type: runSh
+    dependencyMode: strict
+    triggerMode: parallel
+    steps:
+      - IN: v582_u14_x8664_gce_img
+        switch: off
+      - IN: buildami_repo
+        switch: off
+      - IN: ship_bits_gcp
+        switch: off
+      - TASK:
+          runtime:
+            options:
+              env:
+                - BASE_IMG_RES: "v582_u14_x8664_gce_img"
+                - MACHINE_TYPE: "n1-standard-8"
+                - REGION: "us-east1"
+                - ZONE: "us-east1-b"
+                - PROJECT_ID: "ship-bits"
+                - SERVICE_ACCOUNT_JSON: "gcp_key.json"
+                - SSH_USER: "root"
+                - IMG_VER: "v5.8.2"
+          script:
+            - set -o pipefail
+            - |
+               export IMG_VER_DASH=${IMG_VER//./-}
+               export FAM_NAME=$IMG_VER_DASH"-final-patch-u14-x86-64"
+               export SOURCE_IMAGE_FAMILY=$(shipctl get_resource_version_key "$BASE_IMG_RES" "IMAGE_FAM_NAME")
```

Comment:
Using `v582_u14_x8664_gce_img` as source.

Reply:
Also, a new family will be created for this `$IMG_VER_DASH"-final-patch-u14-x86-64`.

Reply:
OK, but we should make sure we delete all traces of this once we are done with the debugging.